api/admin/controller/__init__.py

```diff
@@ -1825,15 +1843,22 @@ def _get_integration_protocols(cls, provider_apis, protocol_name_attr="__module_
 
             settings = getattr(api, "SETTINGS", [])
             protocol["settings"] = list(settings)
+            if _db and issubclass(api, HasIntegrationConfiguration):
+                protocol["settings"] = api.settings_class().configuration_form(_db)
```

Comment:
Both cls.SETTINGS and `def settings_class` must coexist while some integrations still use external integrations.